Lil-Peep/src/styles/tools/mixins.scss

```diff
@@ -0,0 +1,119 @@
+/*================ Mixins ================*/
+@mixin clearfix() {
+  &::after {
+    content: '';
```

Comment:
Properties should be ordered clear, content, display<br>Prefer double-quoted strings